HISTORY.rst

```diff
@@ -8,6 +8,8 @@ History
 * Add "naive" and "top-k" methods in MapieClassifier
 * Include J+aB method in regression tutorial
 * Add MNIST example for classification
+* Add replication of the Chen Xu's tutorial testing J-aB and Jackknife+ on a 
+dataset loaden online.
```

Comment:
You can drop the detail about "dataset loaded online"

Reply:
Done

Reply:
Sorry @tmorzade, you edited the wrong file. This comment was about `HISTORY.rst`, not `plot_kim2020_simulations.py`.

Can you cancel the changes to the latter file and modify `HISTORY.rst` ?

Reply:
Done